src/USER-MISC/pair_agni.cpp

```diff
@@ -68,7 +68,6 @@ static const char cite_pair_agni[] =
   " year      = {2017},\n"
   "}\n\n";
 
-#define AGNI_VERSION 1
```

Comment:
it would be better to use something like
```
enum { AGNI_VERSION_UNKNOWN, AGNI_VERSION_1, AGNI_VERSION_2 };
```
here and then use those constants below instead of "0", "1", and "2" to make it more obvious what is being checked for.